### DISCUSSION TITLE
Steep & curved lift hills from RCT1 for the (Classic) wooden roller coaster

### DISCUSSION THREAD

**Discussion by ZeeMaji** [OpenRCT2/OpenRCT2#18043] [Feature requests]
In RCT1's csg1 there are sprites for steep (used) and curved (unused in normal gameplay but available with trainers) lift hills for the wooden roller coaster, it would be nice if openrct2 could use these sprites if RCT1 is linked especially for the steep ones as we already allow building reverse incline steep lift hills for the wooden coasters for RCT1 parity, they just lack the proper sprites.
![Screenshot_select-area_20220917232214](https://user-images.githubusercontent.com/42477864/190884266-435fad32-7c55-4d82-b9ed-d5c127cba8cb.png)

**Comment by 733737** [comment]
I agree 100% this would be an excellent feature to add to the game!

**Comment by Gymnasiast** [comment]
I don’t think the sprite looks particularly nice. We’d be better off with proper moving chain, especially for the steep down lift hill.

**Reply by ZeeMaji** [reply]
I agree, the wooden coasters in RCT1 don't even actually have the reversed incline shuttle mode and the only reason the steep sprites are visible ingame is due to a weird bug that lets you build steep reversed lift hills on any coaster type that has steep slopes.
The sprites themselves like the curved lift hill sprites are beta leftovers, you can see some very similar looking lift hills sprites in some of the evo of RCT screenshots.
